contents/docs/data/event-spec/ecommerce-events.mdx

```diff
@@ -0,0 +1,1213 @@
+---
+title: Ecommerce Events Specification
+sidebar: Docs
+showTitle: true
+---
+
+PostHog's e-commerce spec helps define the journey for a customer as they browse your store, click on promotions, view products, add those products to a cart, and complete a purchase.
```

Comment:
Can you provide a bit more context here? 
- Where is this used in PostHog? 
- We recommend people capture these events to best work with certain features of PostHog?
- Do any of the libraries autocapture these events?

```suggestion
PostHog's ecommerce spec helps define the journey for a customer as they browse your store, click on promotions, view products, add those products to a cart, and complete a purchase.
```

Reply:
> Where is this used in PostHog?
 
We will base our cdp destination templates on top of this event spec.

> We recommend people capture these events to best work with certain features of PostHog?

Yes, following this event spec, will allow you to easily spin up cdp destinations without having to manually map the correct events or properties to each destination.

> Do any of the libraries autocapture these events?

None of our current SDKs autocapture these or other e-commerce events.

Reply:
Perfect, thanks!